install/kubernetes/cilium/values.yaml

```diff
@@ -1469,7 +1483,7 @@ preflight:
   # -- Annotations to be added to preflight pods
   podAnnotations: {}
 
-  # Labels to be added to preflight pods
+  # -- Labels to be added to thee preflight pod.
```

Comment:
Thou should'st consider speaking  'i  a moe recent English, haply.

Reply:
Forsooth - thanketh thee f'r noticing t!